tsconfig.json

```diff
@@ -4,14 +4,17 @@
       "target": "es2018",
       "module": "commonjs",
       "strict": true,
+      "resolveJsonModule": true,
```

Comment:
this was necessary in order to load the [abi json files](https://github.com/Geo-Web-Project/core-contracts/pull/38/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R1-R8) in index.ts